buildconfig/CMake/LinuxPackageScripts.cmake

```diff
@@ -105,16 +105,19 @@ if ( "${UNIX_DIST}" MATCHES "RedHatEnterprise" OR "${UNIX_DIST}" MATCHES "^Fedor
   if ( "${UNIX_CODENAME}" MATCHES "Santiago" )
     set ( WRAPPER_PREFIX "scl enable mantidlibs34 \"" )
     set ( WRAPPER_POSTFIX "\"" )
-  elseif ("${UNIX_DIST}" MATCHES "^Fedora")
-    # The instrument view doesn't work with the wayland compositor
-    # Override it to use X11 https://doc.qt.io/qt-5/embedded-linux.html#xcb
-    set ( WRAPPER_PREFIX "QT_QPA_PLATFORM=xcb " )
-    set ( WRAPPER_POSTFIX "" )
   else()
     set ( WRAPPER_PREFIX "" )
     set ( WRAPPER_POSTFIX "" )
   endif()
 
+  if ("${UNIX_DIST}" MATCHES "^Fedora")
```

Comment:
The `if` and `else` are doing the same thing. I don't think that is what you were aiming for?